src/interface/update.cpp

```diff
@@ -149,9 +149,10 @@ TaskStatus SparseDealloc(MeshData<Real> *md) {
   const IndexRange kb = md->GetBoundsK(IndexDomain::entire);
 
   auto control_vars = md->GetMeshPointer()->resolved_packages->GetControlVariables();
-  const auto tup = SparsePack<>::Get(md, control_vars, {Metadata::Sparse});
-  auto pack = std::get<0>(tup);
-  auto packIdx = std::get<1>(tup);
+  auto desc = MakePackDescriptor(md->GetMeshPointer()->resolved_packages.get(),
```

Comment:
Probably for a separate PR, but should we look at using sparse packs in the other parts of the built in machinery, instead of legacy packs? We could in fact store the pack descriptors... either in params for an internal parthenon package, or with tricks like static variables, for a bit of a performance boost.

Reply:
Yeah, I think we should just save that for if/when we remove legacy packs. When I first put in sparse packs, I tried to sneak them into update but it didn't seem like people were on board...